pennylane_forest/qpu.py

```diff
@@ -82,6 +82,14 @@ def __init__(self, device, *, shots=1024, active_reset=True, load_qc=True, reado
         self.readout_error = readout_error
 
         self._eigs = {}
+        self.parametric_compilation = kwargs.get("parametric_compilation", True)
+
+        if self.parametric_compilation:
+            self._lookup_table = {}
+            self._parameter_map = {}
+            self._parameter_reference_map = {}
+
+        timeout = kwargs.pop("timeout", None)
```

Comment:
Same comment here as the QVM: both timeout and parametric_compilation should be added to the docstring